### ISSUE TITLE
Optionally output WireGuard config files

### ISSUE THREAD

**Issue by thebalaa** [fractalnetworksco/selfhosted-gateway#19]
Current behavior of `make link ...` is to output a docker-compose service snippet with WireGuard parameters as environment variables.

A welcome convenience would be the ability to output a WireGuard configuration that can be consumed directly by the existing Linux, Windows and macOS WireGuard clients.

**Comment by Dotsch2005** [comment]
Do you mean that an implementation of `make wireguard` or `make wg` creates a link at the gateway and then outputs the client wg0. conf for direct use?

I find this an interesting idea and would give it a try during the week.

**Comment by thebalaa** [comment]
Right, instead of outputting the compose file we would output the WireGuard config file. 

I appreciate your help, let me know if you have any questions.
